routes/user.router.js

```diff
@@ -0,0 +1,121 @@
+const express = require('express')
+const jwt = require('jsonwebtoken')
+const bcrypt = require("bcrypt");
+const router = express.Router();
+const { User } = require('../models/user.model')
+const { authorizedUser } = require('../utils/authorizedUser')
+const secret = process.env['SECRET'];
+
+router.post('/login', async (req, res) => {
+  const { username, password } = req.body
+  try {
+    const user = await User.findOne({ username: username })
```

Comment:
you can simply pass `const user = await User.findOne({ username })` no need to pass username again (ES6)